driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -1323,6 +1323,39 @@ static __always_inline uint16_t store_cgroup_subsys(struct auxiliary_map *auxmap
 	 *
 	 *  cpuset=/path_part1/path_part2\0
 	 */
+
+	/*
+	 *  If kernfs node is still not NULL, it means last path component retrieved
+	 *  (ie: the first one to be printed) was still not '\0' (ie: '/').
+	 *  To avoid sending a cgroup that does not start with '/', manually add it.
+	 *
+	 *  Example:
+	 *  /user.slice/user-1000.slice/user@1000.service/app.slice/app-org.gnome.Terminal.slice/vte-spawn-2f17b2eb-994e-415d-bce0-44c1447d7cd2.scope
+	 *  gets split as:
+	 *  * vte-spawn-2f17b2eb-994e-415d-bce0-44c1447d7cd2.scope\0
+	 *  * app-org.gnome.Terminal.slice\0
+	 *  * app.slice\0
+	 *  * user@1000.service\0
+	 *  * user-1000.slice\0
+	 *  * user.slice\0
+	 * Without the below fix, we would send to userspace "user.slice/user-1000.slice/..." without
+	 * leading '/'.
+	 *
+	 * Counter example with "5" components + the root:
+	 * /user.slice/user-1000.slice/user@1000.service/app.slice/app-org.gnome.Terminal.slice
+	 * gets split as:
+	 * * app-org.gnome.Terminal.slice\0
+	 * * app.slice\0
+	 * * user@1000.service\0
+	 * * user-1000.slice\0
+	 * * user.slice\0
+	 * * \0
+	 * In this case, we won't push the new character, instead we will push the correct string.
+	 */
+	if(kn) {
+		push__new_character(auxmap->data, &auxmap->payload_pos, '/');
```

Comment:
Fix a bug in how modern_bpf sent cgroups, when number of paths component was greater than `MAX_CGROUP_PATH_POINTERS`.
For example, for 
```
/user.slice/user-1000.slice/user@1000.service/app.slice/app-org.gnome.Terminal.slice/vte-spawn-2f17b2eb-994e-415d-bce0-44c1447d7cd2.scope
```
that is 7 paths long (considering first one, ie: root, is returned by the kernel as an empty string), we would have returned to userspace:
```
user.slice/user-1000.slice/user@1000.service/app.slice/app-org.gnome.Terminal.slice/vte-spawn-2f17b2eb-994e-415d-bce0-44c1447d7cd2.scope
```
ie: the leading `/` was missing.